pyproject.toml

```diff
@@ -1,3 +1,84 @@
+[project]
+name = "prairielearn"
+version = "0.0.0"
+readme = "README.md"
+requires-python = ">=3.10"
+classifiers = [
+    "Private :: Do Not Upload"
+]
+
+dependencies = [
+    "ansi2html==1.9.2",
+    "automata-lib==9.1.2",
+    "chevron==0.14.0",
+    "coloraide==4.7.2",
+    "Faker==37.5.3",
+    "jinja2==3.1.6",
+    "lxml==6.0.0",
+    "matplotlib==3.10.3",
+    "networkx==3.4.2",
+    "nltk==3.9.1",
+    "numpy==1.26.4",
+    "openpyxl==3.1.5",
+    "pandas==2.3.1",
+    "Pillow==11.3.0",
+    "Pint==0.24.4",
+    "psutil==7.0.0",
+    "PuLP==3.2.2",
+    "pyarrow==21.0.0",
+    "pycryptodome==3.23.0",
+    "pygments-ansi-color==0.3.0",
+    "Pygments==2.19.2",
+    "pygraphviz==1.14",
+    "pyquaternion==0.9.9",
+    "regex==2025.7.34",
+    "schemdraw==0.21",
+    "scikit-learn==1.7.1",
+    "scipy==1.15.3",
+    "seaborn==0.13.2",
+    "statsmodels==0.14.5",
+    "sympy==1.14.0",
+    "text-unidecode==1.3",
+    "typing-extensions==4.14.1",
+    "tzlocal==5.3.1",
+]
+
+[dependency-groups]
+dev = [
+    "pytest-cov==6.2.1",
+    "pytest-repeat==0.9.4",
+    "pytest-github-actions-annotate-failures==0.3.0",
+    "pytest==8.4.1",
+    "ruff==0.12.7",
+    "types-chevron==0.14.2.20250103",
+    "types-lxml==2025.3.30",
+    "types-Pygments==2.19.0.20250715",
+    "pandas-stubs==2.3.0.250703",
+]
+
+docs = [
+    "jsonschema2md==1.6.1",
+    "mdx_truly_sane_lists==1.3",
+    "mkdocs-d2-plugin==1.6.0",
+    "mkdocs-exclude==1.0.2",
+    "mkdocs-gen-files==0.5.0",
+    "mkdocs-literate-nav==0.6.2",
+    "mkdocs-material==9.6.16",
+    "mkdocs==1.6.1",
+    "mkdocstrings-python==1.16.12",
+    "mkdocstrings==0.30.0",
+    "pymdown-extensions==10.16.1",
+    # mkdocstrings-python will format the docstrings using ruff
+    # https://mkdocstrings.github.io/python/usage/configuration/signatures/#line_length
+    "ruff==0.12.7",
+    "mkdocstrings-python==1.16.12",
+]
+
+# We only use pyproject.toml for the Python dependencies, so we don't need to
+# specify any packages / source files.
+[tool.setuptools]
+packages = []
```

Comment:
Is adding this with an empty array different from omitting it entirely? I'd have a slight preference to omit it if it's not needed.